### PR TITLE
Ease using the setup scripts when pulling in pyd from dub

### PR DESCRIPTION
**Pull request by John-Colvin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,10 +9,16 @@ PyD provides seamless interoperability between Python and the D programming lang
 
 # Usage
 
-To use with dub, either specify the relevant subConfiguration, or run
-`source set_env_vars.sh <your python>` on linux or
-`set_env_vars.bat <your python>` on windows to set the relevant environment variables
-and use the `env` subConfiguration
+To use with dub, either specify the relevant subConfiguration for your python version,
```
Comment (ariovistus) [inline README.md]: could you add this to somewhere suitable in the readthedocs as well? maybe create `docs/source/dub.rst`

Comment (John-Colvin) [reply README.md]: done

Comment (ariovistus) [reply README.md]: actually, could you also add it to the list in index.rst?

Comment (John-Colvin) [reply README.md]: done

**Pull request merged**